app-backend/api/src/main/scala/platform/Routes.scala

```diff
@@ -116,9 +121,9 @@ trait PlatformRoutes extends Authentication
                   updateOrganization(platformId, orgId)
                 }
               } ~
-              delete {
```

Comment:
I'm not sure this definition of the API is the best, but I was torn between this and having separate activate / deactivate endpoints since I need to support both actions.